modules/jaeger-config-storage.adoc

```diff
@@ -0,0 +1,207 @@
+|`http://elasticsearch.<namespace>.svc:9200`
+
+|elasticsearch:
+  nodeCount:
+|Number of Elasticsearch nodes. For high availability use at least 3 nodes. Do not use 2 nodes as “split brain” problem can happen.
+|Integer value.  For example, Proof of concept = 1,
+Minimum deployment =3
+|1
+
+|elasticsearch:
+  redundancyPolicy:
+|Controls number of Elasticsearch replica shards.
+|`ZeroRedundancy`(no replica shards), `SingleRedundancy`(one replica shard), `MultipleRedundancy`(each index is spread over half of the Data nodes), `FullRedundancy` (each index is fully replicated on every Data node in the cluster)
+|
+
+|elasticsearch:
+  resources:
+    requests:
+      cpu:
+|Number of central processing units for requests, based on your environment’s configuration.
+|Specified in cores or millicores (for example, 200m, 0.5, 1).  For example, Proof of concept = 500m,
+Minimum deployment =1
+|1Gi
+
+|elasticsearch:
+  resources:
+    requests:
+      memory:
+|Available memory for requests, based on your environment’s configuration.
+|Specified in bytes (for example, 200Ki, 50Mi, 5Gi). For example, Proof of concept = 1Gi,
+Minimum deployment = 16Gi*
+|500m
+
+|elasticsearch:
+  resources:
+    limits:
+      cpu:
+|Limit on number of central processing units, based on your environment’s configuration.
+|Specified in cores or millicores (for example, 200m, 0.5, 1). For example, Proof of concept = 500m,
+Minimum deployment =1
+|
+
+|elasticsearch:
+  resources:
+    limits:
+      memory:
+|Available memory limit based on your environment’s configuration.
+|Specified in bytes (for example, 200Ki, 50Mi, 5Gi). For example, Proof of concept = 1Gi,
+Minimum deployment = 16Gi*
+|
+
+|elasticsearch:
+  secretname:
+|Name of the secret, for example `jaeger-secret`.
+|
+|N/A
+
+
+|storage:
+  esIndexCleaner:
+    enabled:
+|When using Elasticsearch storage, by default a job is created to clean old traces from the index.  This parameter enables or disables the index cleaner job.
+|`true`/ `false`
+|`true`
+
+|storage:
+ esIndexCleaner:
+  numberOfDays:
+|Number of days to wait before deleting an index.
+|Integer value
+|`7`
+
+|storage:
+ esIndexCleaner:
+  schedule:
+|Defines the schedule for how often to clean the Elasticsearch index.
+|Cron expression
+|"55 23 * * *"
+
+|storage:
+ esRollover:
+  schedule:
+|Defines the schedule for how often to rollover to a new Elasticsearch index.
+|Cron expression
+|'*/30 * * * *'
+|
+3+|{asterisk} Each Elasticsearch node can operate with a lower memory setting though this is *not* recommended for production deployments. For production use, you should have no less than 16Gi allocated to each Pod by default, but preferably allocate as much as you can, up to 64Gi per Pod.
+|===
+
+.Production storage example
+[source,yaml]
+----
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: simple-prod
+spec:
+  strategy: production
+  storage:
+    type: elasticsearch
+    elasticsearch:
+      nodeCount: 1
```

Comment:
Defer to @pavolloffay to comment, but given the docs above mention min nodecount = 3 and memory 16Gi for production, we should probably have this example reflect those values?

Reply:
:+1:  that would be better